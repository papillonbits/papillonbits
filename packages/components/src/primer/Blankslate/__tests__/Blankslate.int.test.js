import { regular, narrow, capped, spacious, large, cleanBackground } from './Blankslate.int.story'

describe('<Blankslate />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })

    test('must match narrow()', () => {
      expect(global.renderToJSON(narrow())).toMatchSnapshot()
    })

    test('must match capped()', () => {
      expect(global.renderToJSON(capped())).toMatchSnapshot()
    })

    test('must match spacious()', () => {
      expect(global.renderToJSON(spacious())).toMatchSnapshot()
    })

    test('must match large()', () => {
      expect(global.renderToJSON(large())).toMatchSnapshot()
    })

    test('must match cleanBackground()', () => {
      expect(global.renderToJSON(cleanBackground())).toMatchSnapshot()
    })
  })
})
