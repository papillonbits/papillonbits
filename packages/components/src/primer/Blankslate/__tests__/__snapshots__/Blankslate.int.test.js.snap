// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blankslate /> Render must match capped() 1`] = `
<div>
  <div
    class="blankslate blankslate-capped"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;

exports[`<Blankslate /> Render must match cleanBackground() 1`] = `
<div>
  <div
    class="blankslate blankslate-clean-background"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;

exports[`<Blankslate /> Render must match large() 1`] = `
<div>
  <div
    class="blankslate blankslate-large"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;

exports[`<Blankslate /> Render must match narrow() 1`] = `
<div>
  <div
    class="blankslate blankslate-narrow"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;

exports[`<Blankslate /> Render must match regular() 1`] = `
<div>
  <div
    class="blankslate"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;

exports[`<Blankslate /> Render must match spacious() 1`] = `
<div>
  <div
    class="blankslate blankslate-spacious"
  >
    <h3>
      This is a blank slate
    </h3>
    <p>
      Use it to provide information when no dynamic content exists.
    </p>
  </div>
</div>
`;
